python/paddle/fluid/contrib/slim/tests/convert_model.py

```diff
@@ -0,0 +1,85 @@
+#   copyright (c) 2019 paddlepaddle authors. all rights reserved.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
+import unittest
+import os
+import sys
+import argparse
+import logging
+import struct
+import six
+import numpy as np
+import time
+import paddle
+import paddle.fluid as fluid
+from paddle.fluid.framework import IrGraph
+from paddle.fluid.contrib.slim.quantization import Qat2Int8MkldnnPass
+from paddle.fluid import core
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--model_path', type=str, default='', help='A path to a model.')
+    parser.add_argument(
+        '--model_save_path',
+        type=str,
+        default='',
+        help='A path to the converted model.')
+    parser.add_argument(
+        '--file_name',
+        type=str,
+        default='',
+        help='A name to save file. Default - name from model path will be used')
+    parser.add_argument(
+        '--pdf',
+        type=bool,
+        default=False,
+        help='Convert dot file to pdf. Default False.')
+
+    test_args, args = parser.parse_known_args(namespace=unittest)
+    return test_args, sys.argv[:1] + args
+
+
+def convert_model(original_path, save_path, file_name, to_pdf):
```

Comment:
The name of the method, the script file, and the test seems a bit misleading to me, as it suggests transformation from one form of a model to another one. What about calling it `generate_dot_for_model` or `generate_graph_for_model`? Regardless of whether a .pdf file is generated or not.